### ISSUE TITLE
Add new "forked-from-repo" field

### ISSUE THREAD

**Issue by tlambertz** [neodyme-labs/solana-security-txt#7]
We could add a new optional "forked-from-repo" field to easily indicate when a project is based on another.

In case a bug is found, this field would allow to easily find related projects, which might have similar security issues.

**Comment by mryalamanchi** [comment]
Would it be helpful if the commit hash (short) is added along with the url and name of the project?

```
forked-from-repo: "name:projectname,url=https://github.com/solana-labs/solana,chash=4e5edd8a461dfa2304d721ebd378ae82b0abb856"
```

Or we can add branch name or release tag, instead of commit hash.

**Comment by saw11677** [comment]
I think this sounds and is a much more efficient and effective way of catching a problem before it happens. It would save time , headaches, money and keep everything transparent to a certain extent.

**Comment by salvatore1923** [comment]
Reembolso

**Comment by Johnson1s** [comment]
ID 73bb5308-d754-4ef2-b058-6b1b90e4cc8c
  Wallet:0xe4025DF65DdF62Bcf234e186484c7302D20FE121@43114

**Comment by salvatoregnazzi** [comment]
> We could add a new optional "forked-from-repo" field to easily indicate when a project is based on another.
> 
> In case a bug is found, this field would allow to easily find related projects, which might have similar security issues.

**Comment by MrG9090** [comment]
No

**Comment by MrG9090** [comment]
forked-from-repo: "name:projectname,url=https://github.com/solana-labs/solana,chash=4e5edd8a461dfa2304d721ebd378ae82b0abb856"

**Comment by bassamawada** [comment]
https://github.com/neodyme-labs/solana-security-txt/issues/7

**Comment by bassamawada** [comment]
/

**Comment by jorditho125-alt** [comment]
Ok

**Comment by jquim1998-code** [comment]
X

**Comment by vcarder24-hue** [comment]
Yes add new fork from repo field. I'm having problems logging into my
account with my new phone. SAMSUNG GALAXY A17.5G
PAUL V CARDER
